models/inh_poisson_generator.h

```diff
@@ -0,0 +1,233 @@
+/*
+ *  inh_poisson_generator.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+/*BeginDocumentation
+  Name: inh_poisson_generator - provides Poisson spike trains at a piecewise
```

Comment:
I am not happy with the name, `inh` is too little informative. I would suggest `inhomogeneous_poisson_generator`,  even though it is rather long, as the very least `inhom_...` to avoid any confusion with "inhibitory". The class name should change accordingly.